src/installation/guides/fde.md

```diff
@@ -1,13 +1,13 @@
 # Full Disk Encryption
 
-Your drive's block device and other information may be different, so make sure
-it is correct.
+**Warning**: Your drive's block device and other information may be different,
+so make *certain* it is correct.
```

Comment:
I think the addition of a bold warning is ok, but please use "make sure" instead of "make certain".